doc/man3/EVP_PKEY_encapsulate.pod

```diff
@@ -102,7 +102,7 @@ Encapsulate an RSASVE key (for RSA keys).
 
 L<EVP_PKEY_CTX_new_from_pkey(3)>,
 L<EVP_PKEY_decapsulate(3)>,
-L<EVP_KEM-RSA(7)>, L<EVP_KEM-X25519(7)>, L<EVP_KEM-EC(7)>
+L<EVP_KEM-RSA(7)>, L<EVP_KEM-X25519(7)>, L<EVP_KEM-EC(7)>, L<EVP_KEM-MLKEM(7)>
```

Comment:
```suggestion
L<EVP_KEM-RSA(7)>, L<EVP_KEM-X25519(7)>, L<EVP_KEM-EC(7)>, L<EVP_KEM-ML-KEM-768(7)>
```

Reply:
Changed to all specific key types in https://github.com/openssl/openssl/pull/26037/commits/5b7aad109796393b11b35d43474b444283ccb698